hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/commit/BaseCommitActionExecutor.java

```diff
@@ -230,10 +230,6 @@ protected String getSchemaToStoreInCommit() {
     return config.getSchema();
   }
 
-  protected boolean isWorkloadProfileNeeded() {
-    return true;
-  }
-
```

Comment:
There is no use case for it to return false right?

Reply:
yes, it always returns true in all code.